ccmlib/scylla_node.py

```diff
@@ -1449,6 +1449,10 @@ def rollback(self, upgrade_to_version):
     def hostid(self, timeout=60, force_refresh=False):
         if self.node_hostid and not force_refresh:
             return self.node_hostid
+        m = self.grep_log("init - Setting local host id to (.+?)$")
```

Comment:
this is for stages we don't have the API available yet ?

Reply:
Not only. Sometimes we want to get host id of a host that is already down.